### PR TITLE
LTM updates for Cookiecutter

### PR DESCRIPTION
**Pull request by smk4664**
This PR includes fixes to get a cookie baked with the ltm cookiecutter to work.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -41,19 +41,20 @@ ipython = "*"
 pydocstyle = "*"
 pylint = "*"
 pylint-django = "*"
-pylint-nautobot = "*"
+pylint-nautobot = "^0.3.1"
```
Comment (gsnider2195) [inline pyproject.toml:44]: Does pylint nautobot support nautobot v1.x?

**Comment by gsnider2195** [comment]
`.github/workflows/ci.yml` needs to be updated to change `env.PLUGIN_NAME` to `nautobot-dev-example`.

File: pyproject.toml (hunk 1 above)
Comment (smk4664) [reply pyproject.toml:44]: Yes sir! 

```
[tool.pylint-nautobot]
supported_nautobot_versions = [
    "1.6.0"
]
```

Comment (gsnider2195) [reply pyproject.toml:44]: I see it now. Thanks!

**Comment by smk4664** [comment]
> `.github/workflows/ci.yml` needs to be updated to change `env.PLUGIN_NAME` to `nautobot-dev-example`.

I will update this and make sure the cookie gets updated as well.

**Comment by gsnider2195** [comment]
> > `.github/workflows/ci.yml` needs to be updated to change `env.PLUGIN_NAME` to `nautobot-dev-example`.
> 
> I will update this and make sure the cookie gets updated as well.

This fix should be in v2.3.1 of the cookie

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**